src/map/battle.cpp

```diff
@@ -6005,6 +6009,8 @@ static int battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list *
 		case ABC_CHAIN_REACTION_SHOT_ATK:
 			skillratio += -100 + 800 + 2550 * skill_lv;
 			skillratio += 15 * sstatus->con;
+			if (sc && sc->getSCE(SC_CHASING))
+				skillratio += 700;
```

Comment:
```suggestion
				skillratio += 700 * skill_lv;
```
skill description suggests "+(Chain Reaction Shot Lv x700)%"